### DISCUSSION TITLE
Any and all "github pages" sites are met with a connection timeout error, is this easily fixable?

### DISCUSSION THREAD

**Discussion by DPowrie** [community/community#127077] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Sorry this might seem very amateurish because I'm a user rather than a programmer so apologies if this is blatantly easy to fix.

I logged onto my PC as usual today and booted up my browser (brave if that's relevant) and from there was able to access every website but one. I use "asb player" which is based upon github pages. I thought this might just be an issue on their side specifically as I haven't found this issue anywhere else, but as I looked online for "website down" information - it seemed perfectly normal.

This led me to looking up the original "pages.github" site itself and found I was met with the same infinite loading wheel and ultimately an "ERR_CONNECTION_TIMED_OUT" message.

I've tried doing basic things like DNS flushing and such but to no success, is there anyone who could help out?

**Comment by basher** [comment]
I have the same issue with 1 of my 2 personal project websites, hosted via Github Pages. 
Website times out, but builds are successful.

https://pages.github.com/ also times out.

EDIT:
Bizarrely, my website works OK via 4G on my phone, but not wifi.

**Comment by mjones07** [comment]
I'm also having similar issues. I'm timing out to every Github Pages site

**Comment by Mieekaserra** [comment]
I'm on Plusnet ISP - also having the same issue.

**Reply by basher** [reply]
I'm also on Plusnet. I've reset my router, checked firewall settings, etc.

Strangely, it's just 1 of my Github Pages websites that doesn't work!

**Reply by Mieekaserra** [reply]
gonna guess we need to phone up and yell at them to reboot their Hamster DNS server again!

**Reply by rhystmills** [reply]
This has affected some people at our organisation, all on Plusnet or BT - which apparently share Openreach infrastructure.

**Comment by Lucasigh** [comment]
I am also getting this problem on BT, The github debug tool gives 100% packet loss for me.

**Comment by EthanJamesBarron** [comment]
Having the same issue here but GitHub's status page isn't reporting anything. I guess they're just not aware

**Reply by psaikido** [reply]
Hopefully this many comments might alert someone. I'm seeing the same from the UK.

**Comment by Sajid78612** [comment]
+1 here I have to use VPN to access it again. I tried on my phone with 5g (with EE) which had the same result as on WiFi (BT).

**Comment by LiquidZulu** [comment]
+1 to @Sajid78612. Seems to be a UK-issue, turned on VPN to Switzerland and I can now access *.github.io

**Comment by LiteBrite82** [comment, marked as answer]
Hi all!

It looks like [Spamhaus](https://www.spamhaus.org/) blocked all 4 of the IP addresses we use for GitHub pages; our team is working to resolve the issue.

**Reply by psaikido** [reply]
At https://check.spamhaus.org/results?query=185.199.108.153 spamhaus are saying that I need to get my ISP to ask them to delist those 4 IPs. My sites are down for 2 or 3 days at this point so I need to know what's being done so I can make a decision to move out or not. What's happening?

**Reply by Paul-Rettey** [reply]
Ditto, we were impacted yesterday (4 June) at about 10am.

**Reply by LiteBrite82** [reply]
Hi All!
This problem has been resolved; please create a new discussion if you are still having similar problems.

**Reply by glncy** [reply]
Hi LiteBrite82. I can't access any github pages here in Philippines. Looks like it came back again.

**Reply by DevenBL** [reply]
they did it again https://check.spamhaus.org/results?query=185.199.108.153
185.199.109.153 has also been listed on https://www.abuseipdb.com/check/185.199.109.153 along with all the other IP's

**Reply by kimabd** [reply]
Hello , same problem , getting timeout error, my page resolves to 185.199.108.153 , please help

**Comment by ThomasProbyn** [comment]
For anyone wanting a quick fix whilst this is being solved - it seems only the IPv4 addresses have been blocked (for now) so adding the [IPv6 addresses](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain) as AAAA records in your domain name's DNS manager appears to bring the site back online for now. It may just be me - I'm a sample size of 1 but it appeared to work!

**Reply by psaikido** [reply]
Lifesaver!

**Reply by Sajid78612** [reply]
This worked, thank you!

**Comment by Paul-Rettey** [comment]
Good morning,

Is there an SLA against this, just to give context our company runs assets for a number of HE partners and this is seriously affecting our users experience so it would be good to get a time estimate against resolution.

**Reply by Paul-Rettey** [reply]
and for context we cannot ask some of our 5000 users to go away and speak to their ISP to fix an infrastructure problem between github and spamhaus

**Reply by ThomasProbyn** [reply]
Re the "speak to your ISP" - I believe this message is aimed at the owner of the IP addresses (ie GitHub). As far as I know, GitHub need to remove the Pages site which is distributing malware, then communicate with Spamhaus to sort it out. I don't think this is aimed at users and they won't have to contact their ISP. For reference, see when this happened the [last time](https://github.com/orgs/community/discussions/106991). In the meantime, if you can, try using the comment I posted earlier about using the IPv6 addresses as that seems to work for now.

**Reply by Paul-Rettey** [reply]
Thanks Thomas, our Devs are speaking to our IT dept, we're recommending as a  temp workaround using a VPN that goes around the UK  which fixes it for the time being.

**Reply by StuartsProjects** [reply]
Having the same problem here too. 

Is there a notification on the status page;

[https://www.githubstatus.com/history](url)

**Comment by stufro** [comment]
It issue appears to be resolved for me. Is anyone else still having issues?

**Reply by Mieekaserra** [reply]
mine is working also - thanks for letting me know :)

**Reply by StuartsProjects** [reply]
Now working here too.

**Reply by secretsixes** [reply]
All seems good!

**Reply by Paul-Rettey** [reply]
Reporting ok here, our users are reporting that it looks to be resolved.

**Comment by glncy** [comment]
I guess it came back again. Working on my 5g data but not on my Internet WiFi. I'm from Philippines.

**Comment by ThevenRexOff** [comment]
I can't access any github pages here in Ecuador :c
![image](https://github.com/user-attachments/assets/54e35bc8-82cb-4c86-8515-c92fcb844c54)
